prime-router/src/main/kotlin/cli/SenderUtilsCommand.kt

```diff
@@ -5,12 +5,105 @@ import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.required
 import gov.cdc.prime.router.FileSettings
 import gov.cdc.prime.router.Organization
+import gov.cdc.prime.router.Sender
 import gov.cdc.prime.router.azure.HttpUtilities
 import gov.cdc.prime.router.common.Environment
 import gov.cdc.prime.router.tokens.Scope
 import gov.cdc.prime.router.tokens.SenderUtils
 import java.io.File
 
+// TODO: https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/8659
+// This is a utility created to help migrate keys to organizations and can be removed after the migration is done
+class MigratePublicKeys : SettingCommand(
+    name = "migratekeys",
+    help = """
+        This is a temporary command to assist in the migration of API keys from senders to the organization.
+        
+        It moves all keys from all senders associated with the organization to the organization
+    """.trimIndent()
+) {
+    private val orgName by option(
+        "--orgName",
+        metavar = "<organization>",
+        help = "Specify the name of the organization to add the key to"
```

Comment:
"to add the key to" doesn't make sense? Isn't it more accurate to say "to update location of keys for" or something?

Reply:
Will update.